### PR TITLE
Typos and small corrections.

### PR DESCRIPTION
**Pull request by jacekschae**
After reading this I can say that you write better docs with one eye 😄

### CODE REVIEW

File: doc/arch/big-picture.md [hunk 1]
```diff
@@ -190,12 +190,12 @@ So a simple event handle could just assoc what we need into the database.
   (assoc-in tx-env [:db ident :completed?] checked))
 ```
 
-Once the event completes the transaction sees that only `ident` was updated. It'll then check which queries used that `ident` earlier when rendering and signal that they need to update.
+Once the event completes the transaction it sees that only `ident` was updated. It'll then check which queries used that `ident` when rendering and signal that they need to be updated.
```
Comment (thheller) [inline doc/arch/big-picture.md]: I guess this is missing a comma? The transaction sees that only ident was updated, not the event.

Comment (jacekschae) [reply doc/arch/big-picture.md]: I think I know what you mean. What about this:

> Once the event completes, the transaction sees that `ident` was updated. It'll then check which queries used that `ident`, when rendering, and signal that they need to be updated.

Comment (thheller) [reply doc/arch/big-picture.md]: Sounds good.

**Comment by thheller** [comment]
Thanks

**Pull request merged**